docker-compose.yaml

```diff
@@ -10,6 +10,7 @@ services:
       CFG_DATABASE:       'postgres://user:pass@db/utask?sslmode=disable'
       CFG_UTASK_CFG:      '{"admin_usernames":["admin"],"application_name":"µTask"}'
       CFG_BASIC_AUTH:     '{"admin":"1234","resolver":"3456","regular":"5678"}'
+      CFG_GROUPS_AUTH:    '{"admin":["admins","resolvers"],"resolver": ["resolvers"]}'
```

Comment:
You missed to update this configuration key. Group name is now the key of this map